app/components/common/Banner/components/Announcements/announcements.tsx

```diff
@@ -25,13 +19,22 @@ export const Announcements = ({
   return (
     <Fragment>
       {isSessionTimeout ? (
-        <SessionTimeout onClose={clearSessionTimeout} />
+        <Banner onClose={clearSessionTimeout}>
```

Comment:
Using the generic `Banner` component and removing unnecessary SessionTimeout, SystemSystem and SystemIndexing components from AnVIL-CMG.